react/src/components/ImageList.tsx

```diff
@@ -197,7 +198,14 @@ const ImageList: React.FC<{ style?: React.CSSProperties }> = ({ style }) => {
             sorter: (a: EnvironmentImage, b: EnvironmentImage) =>
               localeCompare(a?.base_image_name, b?.base_image_name),
             render: (text: string, row: EnvironmentImage) => (
-              <TextHighlighter keyword={imageSearch}>{text}</TextHighlighter>
+              // Replace with this code after using ImageNode instead of Image
+              // <AliasedBaseImageName
+              //   imageFrgmt={row}
+              //   highlightKeyword={imageSearch}
+              // />
+              <TextHighlighter keyword={imageSearch}>
+                {tagAlias(text)}
+              </TextHighlighter>
```

Comment:
Since the base_image_name is simply wrapped in a tagAlias, is there a benefit to leaving it out as a Fragment of the image node?

Reply:
From 24.09.1, we tried to show the server data as much as possible. For the customized item, I made a new component so that we just modified the customized component when necessary.

Reply:
Yes. I thought that the `AliasedBaseImageName` component simply wraps the server data in a tagAlias without any logic like data transformation or navigation, so I didn't think it was necessary to write it as a separate component.